docs/forgeops_doctor_operators.md

```diff
@@ -0,0 +1,59 @@
+## forgeops doctor operators
+
+Verify that operators are installed and ready
+
+### Synopsis
+
+
+	    Checks to ensure that required operators are installed and ready.
+	    
+
+```
+forgeops doctor operators [flags]
+```
+
+### Examples
+
+```
+
+		# check for operators in any namespaces
+		forgeops doctor operators
+		# check for operators in single namespace
+		forgeops doctor operators --all-namespaces=false
+		
+```
+
+### Options
+
+```
+  -A, --all-namespaces   Default: true. If present, list the requested object(s) across all namespaces. Namespace in current context is ignored even if specified with --namespace. (default true)
+  -h, --help             help for operators
+```
+
+### Options inherited from parent commands
+
+```
+      --as string                      Username to impersonate for the operation
+      --as-group stringArray           Group to impersonate for the operation, this flag can be repeated to specify multiple groups.
+      --cache-dir string               Default cache directory (default "/home/max/.kube/cache")
```

Comment:
s/max/user/ or something like that

Reply:
We have a ticket for this... it's kind of annoying - We just run the "cobra" libs doc method, and that's what it's spitting out.